cmd/infra-main/main.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/ako-infra/ingestion"
+	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/lib"
+	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/utils"
+
+	oshiftclient "github.com/openshift/client-go/route/clientset/versioned"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	masterURL  string
+	kubeconfig string
+	version    = "dev"
+)
+
+func main() {
+	InitializeAKOInfra()
+}
+
+func InitializeAKOInfra() {
+	if !lib.IsVCFCluster() {
+		utils.AviLog.Fatalf("Not running in vcf cluster, shutting down")
+	}
+
+	var err error
+	kubeCluster := false
+	utils.AviLog.Info("AKO-Infra is running with version: ", version)
+	// Check if we are running inside kubernetes. Hence try authenticating with service token
+	cfg, err := rest.InClusterConfig()
+	if err != nil {
```

Comment:
is this block of code really necessary? It looks. like we will always run inside the kubernetes cluster?